search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -268,21 +268,27 @@ class TemporalEntityAttributeService(
     ): Mono<List<TemporalEntityAttribute>> {
         val selectQuery =
             """
+                WITH entities AS (
+                    SELECT distinct(entity_id)
```

Comment:
you have to apply the ordering when you get the list of matching entities. if you don't, you'll get unpredictable results.

Reply:
you also need to apply the filters in this query, you only want the list of entities that match the requested filters (type, id pattern, ...)

Reply:
I add the ordering but I need to filters two times

Reply:
It's necessary because when I first filter on my id entities I get a set of id entities. Then I ask for the full temporal entity attribute information with the id of the one included in my set of ids. But for example an attribute that is not wanted can have an id that is in my list so it will be returned.

Reply:
hmm, ok, I see. so maybe the second filtering (on the full TEA) is lighter than the first one (only filtering on `attrs`)?

Reply:
I wonder if it can't have the same problem for the type (multityping)